index.js

```diff
@@ -1,7 +1,13 @@
 const awaitEvent = require('await-event')
 const socketLocation = require('socket-location')
 
-module.exports = async function httpRequestToUrl (request) {
+module.exports = async function httpRequestToUrl (request, opts = {}) {
+  // default to true
+  console.log(opts.followProxies)
+  console.log(!!opts.followProxies)
+  opts.followProxies = opts.followProxies !== false
+  console.log(opts.followProxies)
```

Comment:
Did you forget to remove the `console.log(...)` calls before pushing? This shouldn't log anything.

Reply:
> Did _you_ forget

Me? I'd never do such a thing -- it was clearly one of my cat(s).  🤦

`console.log`'s removed, with my standard suggestion of improving the linting to catch these sorts of things automatically.